core/src/main/scala/org/apache/spark/h2o/converters/ReadConverterContext.scala

```diff
@@ -49,11 +49,7 @@ trait ReadConverterContext {
   def getTimestamp(columnNum: Int): Long  = getLong(columnNum) * 1000L
   def getUTF8String(columnNum: Int): UTF8String = {
     val str = getString(columnNum)
-    if(str == null){
-      null
-    }else{
-      UTF8String.fromString(str)
-    }
+    if(str == null) null else UTF8String.fromString(str)
```

Comment:
this can be changed to `def getUTF8String(columnNum: Int): UTF8String = UTF8String.fromString(getString(columnNum))`